scraping/scraping/start_scripts/zalando_de.py

```diff
@@ -73,38 +93,58 @@ def male() -> List[dict]:
-        "sports-damen-struempfe": ("UNDERWEAR", {"type": "SPORT"}),
-        "sportanzuege-damen": ("TRACKSUIT", {"type": "SPORT"}),
-        "sports-bademode-damen": ("SWIMMWEAR", {"type": "SPORT"}),
-        "sportschuhe-damen": ("SHOES", {"type": "SPORT"}),
-        "sports-taschen-rucksaecke-damen": ("BAG", {"type": "SPORT"}),
+        "/damenbekleidung-kleider/": ProductCategory.DRESS.value,
+        "/t-shirts-kurzarm/": ProductCategory.TSHIRT.value,
+        "/damenbekleidung-tops/": ProductCategory.SHIRT.value,
+        "/damenbekleidung-shirts-poloshirts/": ProductCategory.SHIRT.value,
+        "/langarmshirt/": ProductCategory.SHIRT.value,
+        "/damenbekleidung-blusen-tuniken/": ProductCategory.BLOUSE.value,
+        "/damenbekleidung-pullover-sweater-strickpullover/": ProductCategory.SWEATER.value,
+        "/damenbekleidung-strickjacken/": ProductCategory.JACKET.value,
+        "/damenbekleidung-sweatshirts-hoodies/": ProductCategory.SWEATER.value,
+        "/damenbekleidung-jacken/": ProductCategory.JACKET.value,
+        "/damenbekleidung-jacken-maentel/": ProductCategory.JACKET.value,
+        "/damenbekleidung-jeans/": ProductCategory.JEANS.value,
+        "/damenbekleidung-hosen/": ProductCategory.PANTS.value,
+        "/damenbekleidung-hosen-shorts/": ProductCategory.SHORTS.value,
+        "/damenbekleidung-hosen-overalls-jumpsuit/": ProductCategory.OVERALL.value,
+        "/damenbekleidung-roecke/": ProductCategory.SKIRT.value,
+        "/damenbekleidung-waesche/": ProductCategory.UNDERWEAR.value,
+        "/nachtwaesche/": ProductCategory.NIGHTWEAR.value,
+        "/damenbekleidung-struempfe/": ProductCategory.SOCKS.value,
+        "/damenbekleidung-bademode/": ProductCategory.SWIMWEAR.value,
+        "/sports-damen-shirts-funktion/": (ProductCategory.TSHIRT.value, {"type": "SPORT"}),
+        "/sports-poloshirts-damen/": (ProductCategory.SHIRT.value, {"type": "SPORT"}),
+        "/sports-top-damen/": (ProductCategory.TOP.value, {"type": "SPORT"}),
+        "/sports-langarmshirts-damen/": (ProductCategory.SHIRT.value, {"type": "SPORT"}),
+        "/sports-blusen-damen/": (ProductCategory.BLOUSE.value, {"type": "SPORT"}),
+        "/sports-damen-shirts-trikot/": (ProductCategory.SHIRT.value, {"type": "SPORT"}),
+        "/sports-jacken-damen/": (ProductCategory.JACKET.value, {"type": "SPORT"}),
+        "/sporthosen-damen/": (ProductCategory.PANTS.value, {"type": "SPORT"}),
+        "/sports-damen-pullover-sweater/": (ProductCategory.SWEATER.value, {"type": "SPORT"}),
+        "/sports-kleider-roecke/": (ProductCategory.DRESS.value, {"type": "SPORT"}),
+        "/sport-bh/": (ProductCategory.UNDERWEAR.value, {"type": "SPORT"}),
+        "/funktionsunterwaesche-damen/": (ProductCategory.UNDERWEAR.value, {"type": "SPORT"}),
+        "/sports-damen-struempfe/": (ProductCategory.SOCKS.value, {"type": "SPORT"}),
+        "/sportanzuege-damen/": (ProductCategory.TRACKSUIT.value, {"type": "SPORT"}),
```

Comment:
One more thing: This category includes "Sport-Einteiler & Bodys" which should be assigned to `OVERALL `with `{"type": "SPORT"}`